### ISSUE TITLE
Syntax checking for JSON and converting to Graph format

### ISSUE THREAD

**Issue by RebeccaGreiner** [amosproj/amos2024ss05-knowledge-graph-extractor#24]
## User story
1. As a dev team
2. I want to check the .json of the LLM results
3. So that the data can later be saved properly into the graph DB.

## Acceptance criteria
1. **Input Formats**:
   - Use dummy data provided by Nikolas from the prompt template POC and/or create dummy data similar to industry partner's linked article https://towardsdatascience.com/how-to-convert-any-text-into-a-graph-of-concepts-110844f22a1a
   - LLM JSON results
  

2. **Process Requirements**:
   - Check .json validity
   - Convert the combined data into a NetworkX graph format.

3. **Graph Format**:
   - The output graph must be in GraphML format, suitable for importing into JanusGraph.

---
#### Comments
- GraphML works with the query language gremlin
--- 
## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by borges-filipe** [comment]
After talking with @nikolas-rauscher we decided on the flow:
Syntax checking of string -> into Python Dataframe -> GraphML format
